core/types/deposit_tx.go

```diff
@@ -117,3 +118,10 @@ func (tx *Transaction) From() common.Address {
 	}
 	return tx.inner.(interface{ from() common.Address }).from()
 }
+
+func JovianDepositTx(daFootprintGasScalar uint16) *DepositTx {
```

Comment:
Although I did suggest you promote this function out of a test, I am a bit worried about having it here, especially with no godoc. This function is preparing a stubbed out / test L1 attributes transaction which may make sense in testing (and _maybe_ in simulation, although I am beginning to doubt that) but definitely not in general in production code. See other comment for some other alternative ideas @joshklop suggested.